pkgs/applications/networking/warp/default.nix

```diff
@@ -33,14 +34,23 @@ stdenv.mkDerivation rec {
     hash = "sha256-GRxZ3y1PdJpBDnGCfmOmZgN8n1aaYf9IhyszRwo3MjQ=";
   };
 
+  patches = [
+    # https://gitlab.gnome.org/World/warp/-/merge_requests/74
+    (fetchpatch2 {
+      name = "rust-1.80-compat.patch";
+      url = "https://gitlab.gnome.org/World/warp/-/commit/32649fa6c08980079fed22fe962329497ceb1974.patch";
```

Comment:
Since the MR is unmerged, I think we need to include this patch in Nixpkgs, in case e.g. the MR gets force pushed and GitLab GCs thet commit.  Fortunately it's not very big.